bottery/platform/__init__.py

```diff
@@ -41,8 +41,9 @@ def webhook_endpoint(self):
     def webhook_url(self):
         return 'https://{}{}'.format(settings.HOSTNAME, self.webhook_endpoint)
 
+    @property
+    def tasks(self):
+        return getattr(self, '__tasks__', [])
```

Comment:
Creating `__attrs__` is not a good idea, if you wanted to create a private attr, `_attr` would be a better name. But in this particular case, I believe we could return the list itself. Maybe we don't even need a function or a property, just the class attribute would good enough for now.